config/config.go

```diff
@@ -418,6 +418,26 @@ type Local struct {
 	// features like catchpoint catchup would be rendered completly non-operational, and many of the node inner
 	// working would be completly dis-functional.
 	DisableNetworking bool `version[16]:"false"`
+
+	// ForceFetchTransactions allows to explicitly configure a node to retrieve all the transactions
+	// into it's transaction pool, even if those would not be required as the node doesn't
+	// participate in the consensus or used to relay transactions.
+	ForceFetchTransactions bool `version[17]:"false"`
+
+	// EnableVerbosedTransactionSyncLogging enables the transaction sync to write extensive
+	// message exchange information to the log file. This option is disabled by default,
+	// so that the log files would not grow too rapidly.
+	EnableVerbosedTransactionSyncLogging bool `version[17]:"false"`
+
+	// TransactionSyncDataExchangeRate overrides the auto-calculated data exchange rate between each
+	// two peers. The unit of the data exchange rate is in bytes per second. Setting the value to
+	// zero implies allowing the transaction sync to dynamically calculate the value.
+	TransactionSyncDataExchangeRate uint64 `version[17]:"0"`
+
+	// TransactionSyncSignificantMessageThreshold define the threshold used for a transaction sync
+	// message before it can be used for calculating the data exchange rate. Setting this to zero
+	// would use the default values.
+	TransactionSyncSignificantMessageThreshold uint64 `version[17]:"0"`
```

Comment:
Generally speaking, we attempt to avoid exposing too many default values directly in the config file, especially when the underlying code is rapidly changing. In this particular case, I expect the real default value to change in the coming releases ( once we make the data exchange rate more reliable ). Having a concrete value here means that it's going to be exposed in the config.json file.
What I'd like to avoid is to have an end user tweak with the value - since that would mean that the migration would not take the "newer" version.  Leaving it as 0, allows us to enjoy both worlds - we can have a configurable value now, have it changed later on, and it would be opaque for the end user.